protocol-units/settlement/mcr/setup/src/local/mod.rs

```diff
@@ -107,7 +107,7 @@ impl Local {
 				.ok_or(anyhow!("Failed to get Anvil address"))?
 				.address
 				.clone();
-			info!("hot: deployer address: {}", deployer_address)
+			info!("Deployer address: {}", deployer_address)
```

Comment:
```suggestion
			info!("deployer address: {}", deployer_address)
```

To facilitate matching events by data, consider using KVs:

```
info!(address = %deployer_address, "deployer");
```